transformations/tense/test.json

```diff
@@ -0,0 +1,75 @@
+{
+   "type": "tense_transformation",
+   "test_cases": [
+      {
+         "class": "TenseTransformation",
+         "args": {
+            "to_tense": "past"
+         },
+         "inputs": {
+            "sentence": "I will go to the park."
```

Comment:
Since the transformation contains a lot of logic, can you also add longer sentences too (containing more than 1-verb) and also add cases where the transformation works great and where it does not.

Reply:
Thanks for pointing this out , we have added a few more test cases to help provide a better picture